packages/material/src/mui-controls/MuiInputInteger.tsx

```diff
@@ -40,16 +45,17 @@ export const MuiInputInteger = React.memo(
       handleChange,
       config
     } = props;
-    const inputProps = { step: '1' };
-    const toNumber = (value: string) =>
-      value === '' ? undefined : parseInt(value, 10);
+    const inputProps = useRef({ step: '1' });
```

Comment:
Let's remove the `useRef` here. I'm wondering why this doesn't break anything as this is still given unchanged to `Input`.